src/core/thread/fiber.d

```diff
@@ -605,12 +605,13 @@ class Fiber
           size_t guardPageSize = PAGESIZE ) nothrow
     in
     {
-        assert( dg );
+//        assert( cast(void delegate() const) dg );
+//        assert( dg );
```

Comment:
Why not just delete this code instead of leaving it commented?

Reply:
Because I don't know what is triggering the buildkite failures, or if those failures are unrelated to this PR.

Reply:
The vibe-d and mir-algorithm failures are unrelated and being worked on